v2v-helper/migrate/migrate.go

```diff
@@ -908,11 +924,15 @@ func (migobj *Migrate) cleanup(vminfo vm.VMInfo, message string) error {
 func (migobj *Migrate) CinderManage(rdmDisk vm.RDMDisk) (map[string]interface{}, error) {
 	openstackops := migobj.Openstackclients
 	dat, err := openstackops.CinderManage(rdmDisk)
-	if err != nil {
+	if err != nil || dat == nil {
+		return nil, fmt.Errorf("failed to import LUN: %s", err)
+	}
+	str := dat["id"].(string)
+	if str == "" {
```

Comment:
In the manage operation we can Marshal the response object into OpenStack volume.Volume so that it is easy to work with. 
Note: the actual volume object is inside volume key in the response. And inside the volume object we have ID etc.
```json
{
  "volume": { volume object }
}
````